src/main/java/com/example/kidz_school/entity/Comment.java

```diff
@@ -0,0 +1,26 @@
+package com.example.kidz_school.entity;
+
+import jakarta.persistence.*;
+import lombok.Data;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.UUID;
+
+@Data
+@Entity
+public class Comment {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+
+    private UUID id;
```

Comment:
are you sure that we can configure the UUID like this? I mean the @GeneratedValue annotation. Please do some research